cross/ffmpeg4/Makefile

```diff
@@ -59,10 +58,11 @@ OPTIONAL_DEPENDS += cross/shine
 
 # Define x86asm
 ifeq ($(findstring $(ARCH),$(i686_ARCHS) $(x64_ARCHS)),$(ARCH))
-BUILD_DEPENDS = native/nasm
-NASM_PATH = $(abspath $(PWD)/../../native/nasm/work-native/install/usr/local/bin)
-ENV += AS=$(NASM_PATH)/nasm
-ENV += PATH=$(NASM_PATH):$$PATH
+NASM_BINARY = $(shell which nasm)
+ifeq ($(NASM_BINARY),)
+$(error nasm not found. Please intall NASM assembler)
+endif
+ENV += AS=$(NASM_BINARY)
```

Comment:
Is that test even needed?  I did not bother with that for ffmpeg5-6-7, maybe worth either using it everywhere or I would suggest not at all.